package.json

```diff
@@ -103,83 +103,83 @@
   "resolutions": {
     "@angular-devkit/core/rxjs": "^7.4.0",
     "@angular-devkit/schematics/rxjs": "^7.4.0",
-    "@nrwl/angular/jasmine-marbles": "^0.9.2",
+    "@nx/angular/jasmine-marbles": "^0.9.2",
     "@nrwl/webpack/loader-utils": "^1.4.0",
     "jasmine-marbles/rxjs": "^7.4.0",
     "git-url-parse": "^13.1.0",
     "gonzales-pe-sl/minimist": "^1.2.0"
   },
   "dependencies": {
-    "@angular/common": "~15.2.0",
-    "@angular/core": "~15.2.0",
-    "@angular/platform-browser": "~15.2.0",
+    "@angular/common": "~16.0.5",
+    "@angular/core": "~16.0.5",
+    "@angular/platform-browser": "~16.0.5",
     "rxjs": "^7.4.0",
-    "tslib": "^2.4.1",
-    "zone.js": "^0.13.0"
+    "tslib": "^2.5.3",
+    "zone.js": "~0.13.1"
   },
   "devDependencies": {
     "@actions/core": "^1.10.0",
     "@actions/exec": "^1.1.1",
     "@actions/github": "^5.1.1",
-    "@angular-devkit/architect": "~0.1502.0",
-    "@angular-devkit/build-angular": "~15.2.0",
-    "@angular-devkit/core": "~15.2.0",
-    "@angular-devkit/schematics": "~15.2.0",
-    "@angular-eslint/builder": "~15.2.0",
-    "@angular-eslint/eslint-plugin": "~15.2.0",
-    "@angular-eslint/eslint-plugin-template": "~15.2.0",
-    "@angular-eslint/template-parser": "~15.2.0",
-    "@angular/cli": "~15.2.0",
-    "@angular/common": "~15.2.0",
-    "@angular/compiler": "~15.2.0",
-    "@angular/compiler-cli": "~15.2.0",
-    "@angular/platform-browser": "~15.2.0",
-    "@angular/platform-browser-dynamic": "~15.2.0",
+    "@angular-devkit/architect": "~0.1600.5",
+    "@angular-devkit/build-angular": "~16.0.5",
+    "@angular-devkit/core": "~16.0.5",
+    "@angular-devkit/schematics": "~16.0.5",
+    "@angular-eslint/builder": "~16.0.3",
+    "@angular-eslint/eslint-plugin": "~16.0.3",
+    "@angular-eslint/eslint-plugin-template": "~16.0.3",
+    "@angular-eslint/template-parser": "~16.0.3",
+    "@angular/cli": "~16.0.5",
+    "@angular/common": "~16.0.5",
+    "@angular/compiler": "~16.0.5",
+    "@angular/compiler-cli": "~16.0.5",
+    "@angular/platform-browser": "~16.0.5",
+    "@angular/platform-browser-dynamic": "~16.0.5",
     "@babel/core": "~7.22.0",
     "@babel/preset-typescript": "~7.22.0",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
     "@compodoc/compodoc": "^1.1.19",
-    "@nrwl/angular": "~15.9.0",
```

Comment:
This dep is still installed by yarn, you should still get it from somewhere

Reply:
the package has been renamed to `@nx/angular` and `@nx/angular` has a dependency on `@nrwl/angular`:
```shell
$ yarn why @nrwl/angular
├─ @nx/angular@npm:16.3.2
│  └─ @nrwl/angular@npm:16.3.2 (via npm:16.3.2)
│
├─ @nx/angular@npm:16.3.2 [8556a]
│  └─ @nrwl/angular@npm:16.3.2 (via npm:16.3.2)
│
└─ @nx/angular@npm:16.3.2 [b18b1]
   └─ @nrwl/angular@npm:16.3.2 (via npm:16.3.2)
```